src/sections/Community/Newcomers-guide/index.js

```diff
@@ -10,9 +10,13 @@ import CommunityImage4 from "./community-image4.png";
 import TutorialsTable from "./Tutorials-table";
 import NewcomersMap from "./newcomers-map";
 import JoinCommunity from "../Join-community";
+import longArrow from "./long-arrow.svg";
 
 
 const NewcomersGuide = () => {
+
+  const[hover, setHover] = useState({ hover: false });
```

Comment:
This can be just `const[hover, setHover] = useState(false)` right? Can it can be changed accordingly elsewhere

Reply:
Rookie mistake, fixed it!